group-coordinator/src/test/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntimeTest.java

```diff
@@ -504,12 +518,6 @@ public void testScheduleLoadingWithFailure() {
 
         // Verify that onUnloaded is called.
         verify(coordinator, times(1)).onUnloaded();
-
-        // Verify that the listener is deregistered.
```

Comment:
do we no longer unload because the listener is null?

Reply:
yep.